srcpkgs/gmsl/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gmsl'
+pkgname=gmsl
+version=1.1.8
+revision=1
+noarch=yes
```

Comment:
obsolete, archs=noarch is the correct one. Use xlint please

Reply:
xlint gives me no errors with `noarch=yes`.

Reply:
https://travis-ci.org/void-linux/void-packages/jobs/518807357